consensus/hotstuff/verification/combined_signer_v2_test.go

```diff
@@ -78,6 +82,38 @@ func TestCombinedSignWithDKGKey(t *testing.T) {
 
 	expectedSig := signature.EncodeDoubleSig(stakingSig, beaconSig)
 	require.Equal(t, expectedSig, proposal.SigData)
+
+	// vote should be valid
+	vote, err = signer.CreateVote(block)
+	require.NoError(t, err)
+
+	voteValid, err := verifier.VerifyVote(nodeID, vote.SigData, block)
+	require.NoError(t, err)
+	require.Equal(t, true, voteValid)
+
+	// vote on different bock should be invalid
```

Comment:
For the perspective of the signer, it is important to verify that it generates votes that pass validation. Though, the in the following lines, you verify that the verifier rejects invalid votes. I feel these are tests for the  verifier not the singer. 

_Suggestion:_ move the following lines to `combined_verifier_v2_test.go`

Reply:
I'm using the signer tests to test both modules. 

You are right that we can move this specific unhappy case to verifier, but it's just easy to cover all the happy and unhappy path in one file. And the happy path (verifying a valid vote) has to be in this signer tests, the reason is that we need the signer to create a valid vote. 

So it's just easy to maintain all the cases in one file.